### ISSUE TITLE
Need a way to populate the words we have been already tagged.

### ISSUE THREAD

**Issue by YanLiang1102** [oudalab/fajita#7]
So the schema for the dictionary will be 
(word, code, start date, end date, sure or not, userId, taggingTime)
we should not allow two records have the same word and code in this table, which means we not allow one person to have a same role at different time period.
And if the record in the db is sure, we should pop it up when other user try to tag it next time,
(they can change it if they think the sure answer is not correct)
And if the answer there is not sure, we should still pop it up, but if the new tagging is tagged as sure, it should replace the old not sure one in the db, if the new tagging is also not sure, we can keep the old answer there.

**Comment by ahalterman** [comment]
Agree. If the noun or verb phrase is in the dictionary already, it should auto-populate the box on the right.
